piqe_ocp_lib/tests/resources/test_ocp_operators.py

```diff
@@ -126,7 +126,16 @@ def test_get_package_singlenamespace_channel(self, get_test_objects):
         else:
             logger.warning("The randomly picked package doesn't seem to have a single namespace channel")
 
-
+    
+    def test_get_yaml_from_annotation(self, get_test_objects):
+        yfa_obj = get_test_objects.op_hub_obj
+        channel=yfa_obj.get_yaml_from_annotation("amq-streams-1.6.x") 
+        alm=getattr(channel.currentCSVDesc.annotations, "alm-examples")
+        assert len(alm)>1
```

Comment:
If there was an operator with just a single crd, then this would wrongly fail. I think we should just check that we are getting a non empty list

Reply:
done